### ISSUE TITLE
Enforce TE ordering for multi-echo fMRI data

### ISSUE THREAD

**Issue by tsalo** [bids-standard/bids-2-devel#40]
Currently, individual echos in a multi-echo fMRI sequence can occur in an unspecified order, with the “shortest echo” i.e., the echo with the shortest TE, listed as either the first or last echo in the sequence.

This proposal would standardize echo naming such that `*_echo-1*` is always the shortest echo and all following echos are indexed in sequential order. This would enable parsing multi-echo fMRI data from the file names, without relying on associated metadata. It would also standardize echo ordering across datasets, preventing the confusion of co-existing “little-endian” and “big-endian” echo naming schemes.

Relevant discussion: https://groups.google.com/forum/#!topic/bids-discussion/Utv29uL0Bdc

**Original authors**: @emdupre

**Comment by tsalo** [comment]
I think that this could be extended to all naturally ordered indexable metadata-based entities, including the ones proposed by BEP-001 (e.g., `fa`).

**Comment by jbteves** [comment]
Agree; additionally, standardization of echo ordering would make it easy to enumerate echoes by name and combine data quickly, reduce developer and/or analyst burden.
